user-guide/Standard_Apps/Monitoring_Solutions/Ping_Monitoring/Ping_Monitoring_UI.md

```diff
@@ -100,4 +124,4 @@ On the *Destinations* and *Grid View* pages, a filter pane on the left enables y
 
 To limit the filter pane to an overview of the currently active filters, enable the toggle button in the top-right corner. Disable the toggle button to return to the full view.
 
-In the lower-left corner, you can see the total destinations as well as the destinations that are currently active in your DataMiner System.
+In the lower-left corner, you can see the total destinations as well as the destinations that are currently active in your DataMiner System. Also, a progress bar above those destinations will count down to the next ping command.
```

Comment:
Also, a progress bar above those destinations will count down to the next page refresh.